packages/core/src/Modules/Endpoint/Endpoint.jsx

```diff
@@ -88,9 +88,8 @@ const Endpoint = () => {
 
                 if (!values.server) {
                     errors.server = 'Server is required.';
-                } else if (!/^[\w|\-|.]+$/.test(values.server)) {
```

Comment:
removed that validation as it prevents me and others from using server url containing :port 
(and this is only used internally by devs/qa so really no point in extensive/precise validation)

Reply:
so with couple next updates, it should be possible to run platform completely on mocked data, without QA boxes etc (way way faster and reliable). It is possible now though needs a bit of refinment to make it easy to use.